airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1369,6 +1369,7 @@ class ContactsAllBase(Stream):
     page_filter = "vidOffset"
     page_field = "vid-offset"
     primary_key = "canonical-vid"
+    limit_field = "count"
```

Comment:
The previous implementation was using `limit` instead of `count`. This should have no impact as we pass the default value which is 100